docs/schedule/best-practices.zh_TW.md

```diff
@@ -125,11 +126,11 @@ __例子：__
 
 ### fare_attributes.txt
 
-|字段名稱 |推薦 |
-| --- | --- |
-|所有領域 |如果該字段包含在 `agency.txt` 中，則 `agency_id` 應包含在 `fare_attributes.txt` 中。 |
-| |如果票價系統無法準確建模，請避免進一步混淆並將其留空。 |
-| |包括票價（`fare_attributes.txt` 和 `fare_rules.txt`）並儘可能準確地對其建模。在票價無法準確建模的極端情況下，票價應表示為更貴而不是更便宜，這樣客戶就不會試圖以不足的票價登機。如果絕大多數票價無法正確建模，請不要在提要中包含票價信息。 |
+|字段名稱 | 推薦                                                                                                                                      |
+| --- |-----------------------------------------------------------------------------------------------------------------------------------------|
+|agency_id | 應該包括在內，即使 Feed 中只有一家機構。 （另見建議包括agency_id在[agency.txt](#agencytxt)和[routes.txt](#routestxt) )                                            |
```

Comment:
agency_id x 2, agency.txt, routes.txt not formatted as code
extra space before final closing bracket